crates/node/src/key_events.rs

```diff
@@ -245,6 +247,8 @@ async fn resharing_computation_inner(
             .await?;
             KeyshareData::CkdSecp256k1(res)
         }
+        // Note that once we add the BLS type to dtos_contract::PublicKey this might not be needed
+        _ => unreachable!(),
```

Comment:
I couldn't find a better way to do this, as we need to match with a pair of values, for which some variants are not valid.